source/FormationPattern.cpp

```diff
@@ -0,0 +1,102 @@
+/* FormationPattern.cpp
+Copyright (c) 2019-2022 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "FormationPattern.h"
+
+#include "Angle.h"
+#include "DataNode.h"
+
+#include <cmath>
+
+using namespace std;
```

Comment:
Dont we usuallly have only 1 space between this and the namespace after?

Reply:
Yes, looks like I added 2 newlines too much. I'll remove them.